pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.ruff]
+line-length = 120
+# Enable Pyflakes `E` and `F` codes by default.
+select = [
+    "E", "W",  # see: https://pypi.org/project/pycodestyle
+    "F",  # see: https://pypi.org/project/pyflakes
+    "I",  # see: isort
+]
+ignore = [
+    "E501",
+    "E741",
+]
+ignore-init-module-imports = true
+unfixable = ["F401"]
```

Comment:
I think we can make all rules as fixable in pre-commit hooks

Reply:
Why? In some comment before you complained about having config in precommit...

Reply:
because config file is for both IDE/editor plugin and pre-commit hooks, I think this rule should be unfixable in IDE/editor plugin and fixed in pre-commit hook.

Reply:
This one is the https://github.com/PyGithub/PyGithub/pull/2608

Reply:
No, you need to be consistent